### PR TITLE
Add UI shell for navigation and namespace selection

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Continuation towards https://github.com/tektoncd/dashboard/issues/35
Not the final UI, but it works and provides the required structure.
Design updates can follow at a later date.

Add common UI shell to wrap the main content of the page.
This shell is comprised of a header and primary navigation.

The navigation surfaces the top-level views provided by the
dashboard, as well as the ability to switch namespaces.
Selecting a namespace will filter other views to showing
resources defined in that namespace.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by AlanGreene** [comment]
@steveodonovan can you take a look please? This should be good enough until we have the final designs and allows us to continue with other functionality that relies on namespace selection.

File: src/containers/TaskRuns/TaskRuns.js [hunk 1]
```diff
@@ -76,9 +77,12 @@ export /* istanbul ignore next */ class TaskRunsContainer extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    const { match } = this.props;
+    const { match, namespace } = this.props;
     const { taskName } = match.params;
-    if (taskName !== prevProps.match.params.taskName) {
+    if (
+      taskName !== prevProps.match.params.taskName ||
+      namespace !== prevProps.namespace
```
Comment (steveodonovan) [inline src/containers/TaskRuns/TaskRuns.js]: There isn't any generic way to handle these namespace checks ?

Comment (AlanGreene) [reply src/containers/TaskRuns/TaskRuns.js]: Discussed offline, this is being tracked by https://github.com/tektoncd/dashboard/issues/137

**Comment by AlanGreene** [comment]
@steveodonovan ready for another looks, thanks

**Pull request merged**